feat: Color scheme update

Juan-LukeKlopper opened the pull request:
This PR gives our restake frontend a custom WhisperNode dark theme! Please go through all pages and give me any feedback on things you want to change :)

gh0stdotexe reviewed (changes requested):
Hey JL, reviewed your PR and found a few minor tweaks I'd like to make: 

1. logo - looks like it's still using the old variant of our logo in the site menu, can we update this please? (also check and make sure we're using our new brand icon for the site favicon, and check elsewhere to make sure all old variants are replaced/updated)

2. instead of blue for links, could we maybe do red or some other fun color variant? (could try yellow for those as well). I know blue is typically standard, but it might look cleaner with a different color. thoughts here? 

![restake-updates](https://github.com/WhisperNode/WhisperStake-UI/assets/49853598/734a3bb2-9729-41f6-a2e0-1a621d98b1a5)

Juan-LukeKlopper commented:
Hey @gh0stdotexe! I have updated the logo and link colors with the pushes I added this morning, I also changed the operated by ghost section to just have the whispernode logo at the moment, I think once we have all our services themed and named we can update the footer to link to all our different websites (WhisperBlocks, whisperStake, etc)
